CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Features
 ### Enhancements
 - Set neural-search plugin 3.0.0 baseline JDK version to JDK-2 ([#838](https://github.com/opensearch-project/neural-search/pull/838))
+- Support new knn query parameter expand_nested_docs ([#1008](https://github.com/opensearch-project/neural-search/issues/1008))
```

Comment:
> You may put PR link instead of issues link here. It should be: https://github.com/opensearch-project/neural-search/pull/1013

That is correct. Should be this PR link.

> This is put in Unreleased 3.0 part, just wondering should this feature be part of Unreleased 3.0 or Unreleased 2.x?

Should be under Unreleased 2.x

> Seems like this is new feature, should be it under Features or Enhancements?

Kind of subjective matter. Enhancements might looks fine by me.

Reply:
+1, need to be moved under "2.x enreleased" section

Reply:
OK will update it.